filters/general/filter_5_OISD_Blocklist_Basic/metadata.json

```diff
@@ -1,7 +1,7 @@
 {
-  "filterId": "oisd_basic",
+  "filterId": "oisd_small",
```

Comment:
Please do not change filter ID, it will backfire and disable the list for all people who have it enabled